blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -1,5 +1,7 @@
 import React, { createRef, useCallback, useEffect, useReducer, useState } from "react";
-import { Button, BUTTON_STYLES, BUTTON_TYPES } from "@wpmedia/shared-styles";
+import { RESIZER_APP_VERSION } from "fusion:environment";
+import { Button, Join, Stack } from "@wpmedia/arc-themes-components";
+import Divider from "@wpmedia/arc-themes-components/src/components/divider";
```

Comment:
We shouldn't have to import like this, this is fixed with the following components pull request - https://github.com/WPMedia/arc-themes-components/pull/152